### DISCUSSION TITLE
How to upgrade to newer version?

### DISCUSSION THREAD

**Discussion by akvariefisk** [BoidCMS/BoidCMS#15] [General]
An upgrade tutorial maybe needed.

I am sure that are few steps but just to clarify it further.

Regards
Erik

**Comment by BoidCMS** [comment]
Thanks for the suggestion @akvariefisk.
An upgrade tutorial will soon be published.

**Reply by sysadev** [reply]
Hi @akvariefisk, the upgrade documentation has been published: https://boidcms.github.io/#/upgrade

Thanks for the suggestion.
